src/PrestaShopBundle/Resources/config/services/core/util/database.yml

```diff
@@ -0,0 +1,8 @@
+services:
+  _defaults:
+    public: true
+
+  prestashop.core.util.database.entity_schema_manager:
```

Comment:
As it is a new service, you can use autowrire / autoconfigure here, and make this service private